Dockerfile-scheme

```diff
@@ -0,0 +1,32 @@
+FROM opencog/opencog-dev:cli
+MAINTAINER Enkusellasie Wendwosen <enku@singularitynet.io>
+
+#Run apt-get in NONINTERACTIVE mode
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV HOME /root
+WORKDIR $HOME
+
+RUN git clone https://github.com/enku-io/agi-bio.git
```

Comment:
Change this to point to the main repo on opencog or singnet. Or is there a specific reason for this?

Reply:
Currently this is the only one that's building.